### PR TITLE
ODIM-5079 Remove values from wildcard after deleting server

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by skhan-anjum — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

File: svc-aggregation/system/addaggregationsource_test.go [hunk 1]
```diff
@@ -863,6 +863,10 @@ func mockDeleteActiveRequest(managerAddress string) *errors.Error {
 	return nil
 }
 
+func mockGetAllMatchingDetails(table, pattern string, dbtype common.DbType) ([]string, *errors.Error) {
+	return []string{}, nil
```
Comment (bharath-b-hpe) [inline svc-aggregation/system/addaggregationsource_test.go]: Please add proper body to the function.

File: svc-telemetry/tcommon/common_test.go [hunk 2]
```diff
@@ -108,6 +108,14 @@ func mockGetPluginData(pluginID string) (tmodel.Plugin, *errors.Error) {
 	return plugin, nil
 }
 
+func mockGetResource(Table, key string, dbtype common.DbType) (string, *errors.Error) {
+	return "", nil
```
Comment (bharath-b-hpe) [inline svc-telemetry/tcommon/common_test.go]: Please add proper body to the function.

File: svc-telemetry/tcommon/common_test.go [hunk 3]
```diff
@@ -108,6 +108,14 @@ func mockGetPluginData(pluginID string) (tmodel.Plugin, *errors.Error) {
...
+}
+
+func mockGenericSave(body []byte, table string, key string) error {
+	return nil
```
Comment (bharath-b-hpe) [inline svc-telemetry/tcommon/common_test.go:134]: Please add proper body to the function.

File: svc-telemetry/tcommon/common_test.go (hunk 2 above)
Comment (irappa-pattar) [reply svc-telemetry/tcommon/common_test.go]: added

File: svc-aggregation/system/addaggregationsource_test.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-aggregation/system/addaggregationsource_test.go]: added

**Review by bharath-b-hpe — APPROVED** [review]

**Pull request merged**